### ISSUE TITLE
Is blueberry being deprecated?

### ISSUE THREAD

**Issue by shanmukhateja** [linuxmint/blueberry#128]
Hello,

I heard Cinnamon shipped Blueman in the latest offering. So, does this mean this project will be deprecated and archived in the near future?

**Comment by mtwebster** [comment]
Yes, we're no longer using blueberry - the gnome widget we used (That provided most of the UI) is no longer compatible (in Mint 21/Ubuntu 22.04), and there's been some push for a while now that we should just use blueman.
